src/api/spec/models/user_spec.rb

```diff
@@ -22,11 +22,7 @@
 
     it { expect(user.state).to eq(User::STATES['unconfirmed']) }
 
-    it 'validates uniqueness of login' do
-      invalid_user = build(:user, login: user.login)
-      invalid_user.save
-      expect(invalid_user.errors.full_messages).to eq(['Login is the name of an already existing user.'])
-    end
+    it { is_expected.to validate_uniqueness_of(:login).with_message('is the name of an already existing user.') }
```

Comment:
Just checked it ... works! Very strange!

Anyway, LGTM :+1: 
Merge as soon as travis is done